src/conf.rs

```diff
@@ -143,18 +143,22 @@ pub struct Platform {
     /// - TODO: Document(and check) what does it actually mean on android. Transparent window?
     pub framebuffer_alpha: bool,
 
-    /// When using Wayland, this controls whether to draw the default window decorations.
+    /// On Wayland, the decorations are either drawn by the server or via `libdecor`. If neither is
+    /// available then no decorations will be drawn.
+    #[deprecated]
     pub wayland_use_fallback_decorations: bool,
```

Comment:
I would keep this configurable. Maybe as an enum with options for no csd, ugly csd and libdecor?

Reply:
I would still keep it configurable. Boolean worked when there was just two options: SSD only or SSD + CSD fallback, but now we got options! 

Said this, if you think its too much code to maintain - I am okay to just deprecating the flag and doing SSD/libdecor/no decorations automatically.

Reply:
I do not think borderless should affect a normal ssd/csd fallback logic. For SSD/CSD selecteion I would either have an enum or fully automated, no configuration mechanism.

Reply:
Thanks! What a PR! :heart:

Reply:
Absolutely, lets do it!
I would personally prefer X11WithWaylandFallback, but with this PR I don't really see why not WaylandWithX11Fallback!